bokeh/application/handlers/directory.py

```diff
@@ -168,7 +166,7 @@ def safe_to_fork(self):
 
     # Public methods ----------------------------------------------------------
 
-    def modify_document(self, doc):
+    def modify_document(self, doc: Any) -> Any:
```

Comment:
A better signature would be `modify_document(self, doc: Document) -> None`, where `Document` comes from `bokeh.document`. The return type `None` acts like `void` in other languages and this method clearly doesn't return anything. The single `return` statement is just there to break execution early.